src/pip/_internal/utils/misc.py

```diff
@@ -993,7 +996,11 @@ def call_subprocess(
 
 def write_output(msg, *args):
     # type: (str, str) -> None
-    logger.info(msg, *args)
+    command_logger.info(msg, *args)
+    if args == ():
+        print(msg)
+    else:
+        print(msg % args)
```

Comment:
I would `print()` before logging since printing is more important. Also, you should do this in a way that lets you have one `print()` statement. Like, won't `print(msg % args)` work also if `args` is empty?